wp-content/themes/wisconsinwatch/inc/compat-republication-tracker-tool.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Compatibility functions for the Republication Tracker Tool plugin
+ *
+ * @link https://github.com/INN/republication-tracker-tool
+ * @link https://wordpress.org/plugins/republication-tracker-tool/
+ */
+
+/**
+* Hide the Republication sharing widget on posts that are
+* included in the downloads post type
+*
+* @return bool Whether or not the sharing widget should be hidden
+* @link https://secure.helpscout.net/conversation/1016863827/4711?folderId=2730118 convo leading to why
+* @since Republication Tracker Tool version 1.0.2
+*/
+function remove_republish_button_from_downloads( $hide_republication_widget, $post ){
+	if( true !== $hide_republication_widget ){
+		// if the current post is in this category, return true
```

Comment:
@benlk Everything looks good, but can we update this comment to something like `if the current post has this post type, return true`, just in case someone new tries to edit this function